server/user-bootstrap/shared-utils.js

```diff
@@ -49,10 +55,17 @@ module.exports = {
 				'logout_URL',
 				'primary_blog_url',
 				'meta',
+			],
+			decodeWhitelist = [
+				'display_name',
+				'description',
+				'user_URL'
 			];
 
 		allowedKeys.forEach( function( key ) {
-			user[ key ] = obj[ key ];
+			user[ key ] = obj[ key ] && contains( decodeWhitelist, key )
```

Comment:
Whoa, it took me a while to comprehend, I was worried for a while that there was a comparison with a single `=`
I personally dislike ternary operator because of readability issues, but maybe that's just me